### PR TITLE
fix(PolicyCreate): RHICOMPL-1661 add missing tooltip

### PR DESCRIPTION
**Pull request by skateman**
Screenshot from 2021-04-15 13-53-49

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js [hunk 1]
```diff
@@ -110,7 +112,18 @@ const CreateSCAPPolicy = ({ change, selectedBenchmarkId }) => {
                         );
                     })}
                 </FormGroup>
-                <FormGroup label="Policy type" isRequired fieldId="policy-type">
+                <FormGroup
+                    isRequired
+                    labelIcon={
+                        <Tooltip
```
Comment (vkrizan) [inline src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:118]: Observation: I'd suggest to extract this to its own component.

**Review by vkrizan — APPROVED** [review]
Looks good. Thanks.

**Comment by skateman** [comment]
I have no idea why the test is failing since I moved the tooltip to a component :confused:

File: src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js [hunk 2]
```diff
@@ -46,6 +48,15 @@ query benchmarksAndProfiles {
 }
 `;
 
+const PolicyTooltip = () => (
+    <Tooltip
+        position="right"
+        content="Policy types are OpenSCAP policies that are supported by RHEL.
+        For each version of RHEL, users can create one policy of each type.">
```
Comment (akofink) [inline src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:55]:
```suggestion
        For each major version of RHEL, users can create one policy of each type.">
```

**Review by akofink — APPROVED** [review]
Looks great, just one mock difference.

File: src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
```
Comment (akofink) [inline src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:2]: I believe you need to mock this new addition in the tests to get them to pass.

Comment (skateman) [reply src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:2]: How do I do that?

Comment (akofink) [reply src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:2]: Hmm... maybe this isn't the issue after all. I'll investigate and get back to you.

Comment (akofink) [reply src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:2]: @bastilian ok, I'm not sure what's going on here or why it's only the loading state that has an issue. Any ideas?

Comment (bastilian) [reply src/SmartComponents/CreatePolicy/CreateSCAPPolicy.js:2]: @skateman I believe this is an issue with react-renderer and it not being a real browser. I think I encountered the same issue when introducing the first tooltips into components that still used react-test-renderer. 

This seems like a good opportunity to move these tests to enzyme. Here are the tests using shallow rendering, which should be enough for those: https://gist.github.com/bastilian/a9adba1876f7989c76c3626ff152259e#file-gistfile1-js 

Note: `CreateSCAPPolicy` will need to be exported.

**Comment by skateman** [comment]
@bastilian thanks!

**Pull request merged**